package.json

```diff
@@ -319,7 +319,7 @@
     "url-search-params-polyfill": "^8.1.1",
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#b477452525a3360461fe6b604f15b4a792d87264"
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git"
```

Comment:
Maybe a dumb question: 
>Why does the package.json not include the hash but the yarn.lock does?

Reply:
Per https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/README.md#updating-version:

> After changes to vets-json-schema have been merged into master:
> - update vets-website to point to the latest vets-json-schema version by running `yarn update:schema` and making a PR

The package-json version of `vets-json-schema` gets updated to point to the latest commit by running `yarn update:schema`, a script which grabs the latest commit hash and uses that to update the yarn.lock file